main.py

```diff
@@ -145,6 +145,8 @@ def compress_tar(self, path, target):
         files = os.listdir(path)
         tar = tarfile.open(target, 'w')
         for file in files:
+            if file == ".DS_Store":
```

Comment:
I've got some changes coming in via the dev branch, have you got time to edit this PR? I can add this in directly too if you don't mind.